esmvalcore/cmor/check.py

```diff
@@ -257,12 +257,28 @@ def _check_dim_names(self):
                 except iris.exceptions.CoordinateNotFoundError:
                     try:
                         coord = self._cube.coord(coordinate.standard_name)
-                        self.report_error(
-                            'Coordinate {0} has var name {1} instead of {2}',
-                            coordinate.name,
-                            coord.var_name,
-                            coordinate.out_name,
-                        )
+                        if self._cmor_var.table_type == 'CMIP6' and \
+                           coord.ndim > 1 and \
+                           coord.standard_name in ['latitude', 'longitude']:
+                            self.report_warning(
```

Comment:
This should not cause a warning: using CMIP6 data that follows the CMOR conventions will now result in a huge number of warnings. A debug level log message seems more appropriate. Also, I think this should only be applied if automatic fixes are enabled.

Reply:
What we do if we do not fix it automatically? It is not incorrect by itself, just inconvenient

Reply:
Since it is really just a change for convenience' sake, with valid data before and after the change, how about just a `report_info` (which admittedly doesn't exist at the moment)?

Reply:
> how about just a report_info (which admittedly doesn't exist at the moment)?

Better

Reply:
I think debut would be more appropriate, because using info will result in a very large number of log messages, e.g. if you have tens of models with a few variables in a recipe, you will get a hundred or so identical messages printed to your screen.

Reply:
So not only adding a `report_info`, but also a `report_debug` and using the latter here? Sounds good to me.

Reply:
That is done